<launch>
    <node pkg="wpb_home_behaviors" type="wpb_home_objects_3d" name="wpb_home_objects_3d" output="screen">
        <param name="topic" type="string" value="/kinect2/sd/points" />
    </node>
</launch>